### ISSUE TITLE
[Question] Game Package Registry

### ISSUE THREAD

**Issue by CoderGamester** [firebase/quickstart-unity#990]
- Unity editor version: 2020.2.4
- Firebase Unity SDK version: 7.1.0
- Source you installed the SDK:UPM
- Problematic Firebase Component: Crashlytics
- Other Firebase Components in use: Analytics
- Additional SDKs you are using: AppsFlyer
- Platform you are using the Unity editor on: Mac & Windows
- Platform you are targeting: iOS, Android
- Scripting Runtime: IL2CPP

I'm following github and would love to ask for an update on the UPM issue.
I see that was removed from Firebase setup instructions.
As mentioned by @uchar , keeping things in the UPM is so much better for all possible reasons.
I found and follow the link https://laptrinhx.com/managing-the-firebase-unity-plugin-the-easy-way-1600281350/ to configure in my project.
I'm at the moment having some issues having Firebase running on the mobile devices. But without proper instructions on the Firebase documentation is hard to understand where the issue is.

**My question is, isn't this registry made by Google?**

```
Name: Game Package Registry by Google
URL: https://unityregistry-pa.googleapis.com
Scope(s): com.google
```

I can see there the latest updates of the Firebase.

**Comment by chkuang-g** [comment]
@CoderGamester 

Yes, this registry is made and maintained by Google.

Let me know if you have any additional question.

**Comment by CoderGamester** [comment]
You are right.
It seems the issue I was having was trying to build the App with Armv7 & Arm64 was creating issues.
By completely cleanning up the Firebase plugins and adding it back again we were able to make Firebase Analytics work for both iOS & Android.
We decided to drop the use of Crashlytics to avoid any further complications and use Unity's solution for the moment.
Thank you for the indication
